### PR TITLE
Pass command line arguments as flags

### PR DESCRIPTION
**Pull request by Vuksan**
# Description

Modified these commands to have mandatory flags passed as flags:

- peers add
- peers status
- ibft init

# Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have tested this code
- [x] I have updated the README and other relevant documents (guides...)
- [x] I have added sufficient documentation both in code, as well as in the READMEs

# Additional comments

Will update the docs as well.
_EDIT:_ Docs update PR: https://github.com/0xPolygon/polygon-sdk-docs/pull/6

### CODE REVIEW

**Review by lazartravica — APPROVED** [review]
LGTM

File: command/peers_status.go [hunk 1]
```diff
@@ -20,8 +35,9 @@ func (p *PeersStatus) GetHelperText() string {
 // Help implements the cli.PeersStatus interface
 func (p *PeersStatus) Help() string {
 	p.Meta.DefineFlags()
+	p.DefineFlags()
 
-	usage := "peers status PEER_ID"
+	usage := "peers status --libp2p-node-id PEER_ID"
```
Comment (zivkovicmilos) [inline command/peers_status.go]: Kind of a nitpick, but I'm trying to think of a different way we can name this flag.
**libp2p-node-id** just seems a bit long, especially for a command that just returns basic peer data.

What you you think about just **peer-id**?
@Kourin1996 @lazartravica

**Review by zivkovicmilos — APPROVED** [review]
Great and clean, as always 💯

**Review by Kourin1996 — APPROVED** [review]
LGTM. It works locally.

File: command/peers_status.go (hunk 1 above)
Comment (Kourin1996) [reply command/peers_status.go]: I think **peer-id** sounds ok. Can understand fully enough. It's better to change help message also if you change to **peer-id**.

Comment (Vuksan) [reply command/peers_status.go]: My intention was to add more context to the flag itself, but I agree, the command is often used, so the flag should be shorter. Will modify.

**Pull request merged**